src/test/components/AppLocalizationProvider.test.js

```diff
@@ -84,9 +84,9 @@ describe('AppLocalizationProvider', () => {
     expect(getDirection(getState())).toEqual('ltr');
   });
 
-  it('renders its children', async () => {
+  it(`does not render children until it's ready`, async () => {
```

Comment:
small nit: The new title of the test feels like it's only about preventing the render until it's ready and not also about the rendering the children. I wonder if changing it to something like this would be a bit clearer: `renders the children once it's ready`. But current title also works for me.

Reply:
In my mind, these sentences are equivalent :-) but I can do that still.